src/pages/_id/index.vue

```diff
@@ -245,6 +248,19 @@ export default {
     userAvatar() {
       return this.userInfo && this.userInfo.avatar;
     },
+    sortedOwnedNFTClassId() {
+      const sorted = [...this.ownedNFTClassId];
+      sorted.sort((a, b) => {
+        const aIsWritingNFT = isWritingNFT(this.getNFTClassMetadataById(a))
+          ? 1
+          : 0;
+        const bIsWritingNFT = isWritingNFT(this.getNFTClassMetadataById(b))
+          ? 1
+          : 0;
```

Comment:
I think `Number(true) === 1` and `Number(false) === 0` so maybe just use `Number()`